sympy/physics/_biomechanics/musculotendon.py

```diff
@@ -612,3 +978,315 @@ def __repr__(self):
+    >>> activation.x
+    Matrix([[a_muscle(t)]])
+    >>> activation.r
+    Matrix([[e_muscle(t)]])
+    >>> activation.p
+    Matrix([
+    [tau_a_muscle],
+    [tau_d_muscle],
+    [    b_muscle]])
+    >>> activation.rhs()
+    Matrix([[((1/2 - tanh(b_muscle*(-a_muscle(t) + e_muscle(t)))/2)*(3*a_muscle(t)/2 + 1/2)/tau_d_muscle + (tanh(b_muscle*(-a_muscle(t) + e_muscle(t)))/2 + 1/2)/(tau_a_muscle*(3*a_muscle(t)/2 + 1/2)))*(-a_muscle(t) + e_muscle(t))]])
+
+    The musculotendon class requires symbols or values to be passed to represent
+    the constants in the musculotendon dynamics. We'll use SymPy's ``symbols``
+    function to create symbols for the maximum isometric force ``F_M_max``,
+    optimal fiber length ``l_M_opt``, tendon slack length ``l_T_slack``, maximum
+    fiber velocity ``v_M_max``, optimal pennation angle ``alpha_opt, and fiber
+    damping coefficient ``beta``.
+
+    >>> F_M_max = symbols('F_M_max', real=True)
+    >>> l_M_opt = symbols('l_M_opt', real=True)
+    >>> l_T_slack = symbols('l_T_slack', real=True)
+    >>> v_M_max = symbols('v_M_max', real=True)
+    >>> alpha_opt = symbols('alpha_opt', real=True)
+    >>> beta = symbols('beta', real=True)
+
+    We can then import the class ``MusculotendonDeGroote2016`` from the
+    biomechanics module and create an instance by passing in the various objects
+    we have previously instantiated. By default, a musculotendon model with
+    rigid tendon musculotendon dynamics will be created.
+
+    >>> from sympy.physics._biomechanics import MusculotendonDeGroote2016
+
+    >>> rigid_tendon_muscle = MusculotendonDeGroote2016(
+    ...     'muscle',
+    ...     pathway,
+    ...     activation,
+    ...     tendon_slack_length=l_T_slack,
+    ...     peak_isometric_force=F_M_max,
+    ...     optimal_fiber_length=l_M_opt,
+    ...     maximal_fiber_velocity=v_M_max,
+    ...     optimal_pennation_angle=alpha_opt,
+    ...     fiber_damping_coefficient=beta,
+    ... )
+
+    We can inspect the various properties of the musculotendon, including
+    getting the symbolic expression describing the force it produces using its
+    ``force`` attribute.
+
+    >>> rigid_tendon_muscle.force
+    -F_M_max*(beta*(-l_T_slack + Abs(q(t)))*sign(q(t))*Derivative(q(t), t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)) + a_muscle(t)*FiberForceLengthActiveDeGroote2016(sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)/l_M_opt, c_0_fl_M_act, c_1_fl_M_act, c_2_fl_M_act, c_3_fl_M_act, c_4_fl_M_act, c_5_fl_M_act, c_6_fl_M_act, c_7_fl_M_act, c_8_fl_M_act, c_9_fl_M_act, c_10_fl_M_act, c_11_fl_M_act)*FiberForceVelocityDeGroote2016((-l_T_slack + Abs(q(t)))*sign(q(t))*Derivative(q(t), t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)), c_0_fv_M, c_1_fv_M, c_2_fv_M, c_3_fv_M) + FiberForceLengthPassiveDeGroote2016(sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)/l_M_opt, c_0_fl_M_pas, c_1_fl_M_pas))*cos(alpha_opt)
```

Comment:
This seems to work:
```diff
diff --git a/sympy/physics/_biomechanics/musculotendon.py b/sympy/physics/_biomechanics/musculotendon.py
index 0b9a3d54ca..e2128814b6 100644
--- a/sympy/physics/_biomechanics/musculotendon.py
+++ b/sympy/physics/_biomechanics/musculotendon.py
@@ -1077,7 +1077,20 @@ class MusculotendonDeGroote2016(MusculotendonBase):
     ``force`` attribute.
 
     >>> rigid_tendon_muscle.force
-    -F_M_max*(beta*(-l_T_slack + Abs(q(t)))*sign(q(t))*Derivative(q(t), t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)) + a_muscle(t)*FiberForceLengthActiveDeGroote2016(sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)/l_M_opt, c_0_fl_M_act, c_1_fl_M_act, c_2_fl_M_act, c_3_fl_M_act, c_4_fl_M_act, c_5_fl_M_act, c_6_fl_M_act, c_7_fl_M_act, c_8_fl_M_act, c_9_fl_M_act, c_10_fl_M_act, c_11_fl_M_act)*FiberForceVelocityDeGroote2016((-l_T_slack + Abs(q(t)))*sign(q(t))*Derivative(q(t), t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)), c_0_fv_M, c_1_fv_M, c_2_fv_M, c_3_fv_M) + FiberForceLengthPassiveDeGroote2016(sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + Abs(q(t)))**2)/l_M_opt, c_0_fl_M_pas, c_1_fl_M_pas))*cos(alpha_opt)
+    -F_M_max*(beta*(-l_T_slack + Abs(q(t)))*sign(q(t))*Derivative(q(t),
+    t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack +
+    Abs(q(t)))**2)) +
+    a_muscle(t)*FiberForceLengthActiveDeGroote2016(sqrt(l_M_opt**2*sin(alpha_opt)**2
+    + (-l_T_slack + Abs(q(t)))**2)/l_M_opt, c_0_fl_M_act, c_1_fl_M_act,
+    c_2_fl_M_act, c_3_fl_M_act, c_4_fl_M_act, c_5_fl_M_act, c_6_fl_M_act,
+    c_7_fl_M_act, c_8_fl_M_act, c_9_fl_M_act, c_10_fl_M_act,
+    c_11_fl_M_act)*FiberForceVelocityDeGroote2016((-l_T_slack +
+    Abs(q(t)))*sign(q(t))*Derivative(q(t),
+    t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack +
+    Abs(q(t)))**2)), c_0_fv_M, c_1_fv_M, c_2_fv_M, c_3_fv_M) +
+    FiberForceLengthPassiveDeGroote2016(sqrt(l_M_opt**2*sin(alpha_opt)**2 +
+    (-l_T_slack + Abs(q(t)))**2)/l_M_opt, c_0_fl_M_pas,
+    c_1_fl_M_pas))*cos(alpha_opt)
 
     When we created the musculotendon object, we passed in an instance of an
     activation dynamics object that governs the activation within the
```

Reply:
I agree. I've shortened with ellipses and removed a couple of statements that weren't adding any value to the documentation.